### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by SataQiu**
fix some typos

### CODE REVIEW

File: static/docs/reference/generated/kubernetes-api/v1.11/index.html [hunk 1]
```diff
@@ -28954,7 +28954,7 @@ <H2 id="quantity-resource-core">Quantity resource core</H2>
 
 Non-canonical values will still parse as long as they are well formed, but will be re-emitted in their canonical form. (So always use canonical form, or don&#39;t diff.)
 
-This format is intended to make it difficult to use these numbers without writing some sort of special handling code in the hopes that that will cause implementors to also use a fixed point implementation.</P>
+This format is intended to make it difficult to use these numbers without writing some sort of special handling code in the hopes that will cause implementors to also use a fixed point implementation.</P>
```
Comment (xiangpengzhao) [inline static/docs/reference/generated/kubernetes-api/v1.11/index.html]: This and below file are generated.

**Comment by SataQiu** [comment]
@xiangpengzhao Sorry for missing that.  I have removed the changes of the generated files.

**Pull request merged**